.circleci/config.yml

```diff
@@ -208,8 +185,6 @@ commands:
           name: Check for any unrecorded changes in our dependency trees
           command: |
             pip3 install --require-hashes -r ./tools/requirements.txt
-            rustup install nightly
-            rustup default nightly
```

Comment:
Similarly, I don't think these have been actually running on nightly since #3910 landed. So I presume they no longer require nightly and it can be removed.